packages/diagrams/frontend/sirius-components-diagrams/src/sprotty/views/ParametricSVGImageView.tsx

```diff
@@ -0,0 +1,106 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+/** @jsx svg */
+/** @jsxRuntime classic */
+import { RectangularNodeView, svg } from 'sprotty';
+import { createResizeHandles } from './ViewUtils';
+const preventRemovalOfUnusedImportByPrettier = svg !== null;
+
+/**
+ * The view used to display nodes with a svg image style.
+ *
+ * @lfasani
+ */
+export class ParametricSVGImageView extends RectangularNodeView {
+  // @ts-ignore
+  render(node: Readonly<Node>, context: RenderingContext) {
```

Comment:
You need to import the type `Node` because here, you are using the `Node` type from the DOM standard library

Reply:
done